### ISSUE TITLE
Deploy multiple micro-frontend into single static web apps

### ISSUE THREAD

**Issue by robersonliou** [Azure/static-web-apps#823]
Hi, 

Can I deploy multiple micro-frontend into single static web apps?

For example, Suppose my static web apps url is https://my.azurestaticapps.net/, and I hope my micro-frontend-apps can mapping to different sub route in static web apps.

- myapp1 -> https://my.azurestaticapps.net/app1/
- myapp2 -> https://my.azurestaticapps.net/app2/
- myapp3 -> https://my.azurestaticapps.net/app3/

When I deploy only app1, it's perfect.
But after app2 deployment, app1 content was disappear.

Is there any way to keep all the files preserve when multi deployment?
Thanks.

**Comment by mkarmark** [comment]
HI @robersonliou, so basically are you saying there will be 3 separate repos in this setup with micro apps behind one "static web app"?

**Comment by robersonliou** [comment]
> HI @robersonliou, so basically are you saying there will be 3 separate repos in this setup with micro apps behind one "static web app"?

Yes, there will be 3 repos with unique pipeline, but deploy to single static web apps.

**Comment by rapitkan** [comment]
Any progress on this?

**Comment by sivasankar-thalavai** [comment]
any update on this?

**Comment by miguelEsteban** [comment]
What alternatives exist to be able to deploy micro frontend in Azure?
only storage + CDN?

**Comment by rahu431** [comment]
Im planning to use azure static web app for my micro frontend any update on this?
